source/js/admissions.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-undef */
+/* eslint-disable linebreak-style */
+
+// import { ModalHandler, ModalViewer, ModalContainer } from './modules/Modal.js';
```

Comment:
why the comment?

Reply:
Because I wanted to use Javascript built in function of exporting and importing from other Javascript files, however I was stopped by the CORS policy, even though the material and request have the same origin.